packages/mobile/package.json

```diff
@@ -45,6 +45,7 @@
     "@celo/react-native-sms-retriever": "git+https://github.com/celo-org/react-native-sms-retriever#b88e502",
     "@celo/utils": "0.1.20-dev",
     "@react-native-community/async-storage": "^1.9.0",
+    "@react-native-community/clipboard": "git+https://github.com/gnardini/clipboard#01d1c13",
```

Comment:
I opened a PR on their repo: https://github.com/react-native-community/clipboard/pull/73 but I'm not sure they'll accept it since I didn't implement the whole new API, just the method we needed. Should I move the fork to celo-org?